dms/models/dms_file.py

```diff
@@ -84,7 +84,7 @@ class File(models.Model):
         relation="dms_file_tag_rel",
         column1="fid",
         column2="tid",
-        domain="['|', ('category_id', '=', False),('category_id', '=?', category_id)]",
+        # domain="['|', ('category_id', '=', False),('category_id', '=?', category_id)]",
```

Comment:
This would not be correct, domain must be kept or removed by adding a comment in the code explaining why it is not necessary now.

Reply:
It is a good point. I fixed it. Thanks!